app/Resources/views/line/blog_post.html.twig

```diff
@@ -0,0 +1,40 @@
+<div class="blog-post-line">
+    <div class="blog-post-line-border">
+        {% set authorId = ez_field_value(content, 'author') %}
+        {{ render_esi(controller('ez_content:viewAction', {
+            'contentId': authorId,
+            'viewType': 'parts'
+        })) }}
+
+        <div class="blog-post-line-date">
+            {{ ez_field_value(content, 'published_date')|date('M d, Y') }}
+        </div>
+
+        {% if not ez_is_field_empty(content, 'listing_image') %}
+            <div class="blog-post-line-thumbnail">
+                <a href="{{ path(location) }}">
+                    {{ ez_render_field(content, 'listing_image', {
+                        'parameters': {
+                            'alias': 'blog_post_thumbnail',
+                            'class': 'img-responsive'
+                        }
+                    }) }}
+                </a>
+            </div>
+        {% endif %}
+
+        <h2>
+            <a href="{{ path(location) }}">
+                {{ ez_render_field(content, 'title') }}
+            </a>
+        </h2>
+
+        <div class="blog-post-line-intro">
+            {{ ez_render_field(content, 'intro') }}
```

Comment:
Can't into also be a link? Then you'd get rid of extra `<a href="{{ path(location) }}">`s and wrap whole post in one `<a>`